sdk/deviceupdate/Azure.IoT.DeviceUpdate/src/GlobalSuppressions.cs

```diff
@@ -1,12 +1,6 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-// This file is used by Code Analysis to maintain SuppressMessage
-// attributes that are applied to this project.
-// Project-level suppressions either have no target or are given
-// a specific target and scoped to a namespace, type, member, etc.
-
 using System.Diagnostics.CodeAnalysis;
 
-[assembly: SuppressMessage("Usage", "AZC0001:Use one of the following pre-approved namespace groups (https://azure.github.io/azure-sdk/registered_namespaces.html): Azure.AI, Azure.Analytics, Azure.Communication, Azure.Data, Azure.DigitalTwins, Azure.Iot, Azure.Learn, Azure.Media, Azure.Management, Azure.Messaging, Azure.Search, Azure.Security, Azure.Storage, Azure.Template, Azure.Identity, Microsoft.Extensions.Azure", Justification = "<Pending>", Scope = "namespace", Target = "~N:Azure.IoT.DeviceUpdate")]
-[assembly: SuppressMessage("Usage", "AZC0001:Use one of the following pre-approved namespace groups (https://azure.github.io/azure-sdk/registered_namespaces.html): Azure.AI, Azure.Analytics, Azure.Communication, Azure.Data, Azure.DigitalTwins, Azure.Iot, Azure.Learn, Azure.Media, Azure.Management, Azure.Messaging, Azure.Search, Azure.Security, Azure.Storage, Azure.Template, Azure.Identity, Microsoft.Extensions.Azure", Justification = "<Pending>", Scope = "namespace", Target = "~N:Azure.IoT.DeviceUpdate.Models")]
+[assembly: SuppressMessage("Usage", "AZC0016:Invalid ServiceVersion member name.", Justification = "<Pending>", Scope = "type", Target = "~T:Azure.IoT.DeviceUpdate.DeviceUpdateClientOptions.ServiceVersion")]
```

Comment:
Please provide a justification for this suppression.  We will need to make sure this is removed when the library GA's -- would you mind filing an issue to track this?

Reply:
I will file an issue to track all those requirement for GA.

Reply:
add justification